Labber_util.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
File names should be in `snake_case` or `CamelCase` (only for classes), but not both.